src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -1,16 +1,94 @@
-import React from "react";
-import "twin.macro";
+import React, { useRef, useState } from "react";
+import tw from "twin.macro";
 import "styled-components/macro";
 import { useHistory } from "react-router-dom";
 import CqlLibrary from "../../models/CqlLibrary";
+import { Button } from "@madie/madie-components";
+import CreatVersionDialog from "../createVersionDialog/CreateVersionDialog";
+import useCqlLibraryServiceApi from "../../api/useCqlLibraryServiceApi";
+import CreatDraftDialog from "../createDraftDialog/CreateDraftDialog";
 
-export default function CqlLibraryList(props: {
-  cqlLibraryList: CqlLibrary[];
-}) {
+const ErrorAlert = tw.div`bg-red-200 rounded-lg py-3 px-3 text-red-900 mb-3`;
```

Comment:
we don't have these in design systems?

Reply:
I removed the ErrorAlert, instead I added the snackBar from material UI